packages/core/src/components/tag/tag.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component, Host, Prop, h } from '@stencil/core'
+import { IconProps } from '../../icons'
+
+@Component({
+  tag: 'atom-tag',
+  styleUrl: 'tag.scss',
+  shadow: true,
+})
+export class AtomTag {
+  @Prop() color: 'success' | 'danger' | 'warning' | 'dark' | 'light' | 'neutral'
+  @Prop() icon?: IconProps
+  @Prop() customBackgroundColor: string
+  @Prop() customTextColor: string
+
+  render() {
+    return (
+      <Host>
+        <ion-badge
+          color={this.color}
+          style={{
+            backgroundColor: this.customBackgroundColor,
+            color: this.customTextColor,
```

Comment:
(question) are we using `style`, so css variables are working?

Reply:
@felipefialho yes, they are working. Had to use `getComputedStyle` method to translate the var to the CSS value https://github.com/juntossomosmais/atomium/pull/261/commits/96c9b91eab97dd6ded7e2fe77a5bf629cf4fb0db